tests/rules/method.js

```diff
@@ -201,6 +201,10 @@ eslintTester.run("method", rule, {
         { // basic support for SequenceExpressions, which always return the last item - fixes #113
             code: "let a = (0,1,2,34);",
             parserOptions: { ecmaVersion: 6 },
+        },
+        { // Regression test for #124
+            code: "(e = this.n[n.i])(i, r)",
```

Comment:
Can you expand by using an example that calls an "unsafe" function but with a safe parameter, to ensure that the tested code goes deeper? E.g. `"(e = node.insertAdjacentHtml('afterend', '<s>hello</s>')"`

Reply:
:+1: I added an additional case for that scenario.